simulate-event.js

```diff
@@ -1,20 +1,19 @@
-var extend = require('xtend')
 
 /**
  * Set some default options.
  *
  * @type {Object}
  */
 var eventOptions = {
-  UIEvent: function (el) {
+  UIEvent: function () {
     return {
-      view: el.ownerDocument.defaultView
+      view: document.defaultView
```

Comment:
Any particular reason to change this? I'm sure I would have done it this way for a reason 😄 Perhaps I was testing across `iframe` at that point or something else.